rust/syntect-plugin/src/main.rs

```diff
@@ -206,14 +206,19 @@ impl<'a> PluginState<'a> {
         // don't touch indentation if this is not a simple edit
         if end != start { return None }
 
-        let is_newline = &ctx.get_config().line_ending == text;
+        let line_ending = ctx.get_config().line_ending.clone();
```

Comment:
Not sure if this is the right thing to do but I was not able to access &ctx within if due to an error. 
```
closure requires unique access to `ctx` but `*ctx` is already borrowed
```
I am not sure if there is a better way of doing this.

Reply:
Yep this is reasonable.